### PR TITLE
Parse <md> tag in Markdown as inline

### LINKED ISSUE #1591: se-book: Some MarkBind syntax not rendered correctly
Code: https://github.com/damithc/se-book

image

### PR DESCRIPTION
**Pull request by wxwxwxwx9**
**What is the purpose of this pull request?**

- [ ] Documentation update
- [x] Bug fix
- [ ] Feature addition or enhancement
- [ ] Code maintenance
- [ ] Others, please explain:

Resolves #1591. 

**Overview of changes:**
Added `<md>` tag to inline whitelist rule.

**Anything you'd like to highlight / discuss:**
-

**Testing instructions:**
`npm run test`

**Proposed commit message: (wrap lines at 72 characters)**
Parse `<md>` tag in Markdown as inline.

---

**Checklist:** :ballot_box_with_check:

- [ ] Updated the documentation for feature additions and enhancements
- [ ] Added tests for bug fixes or features
- [x] Linked all related issues
- [x] No blatantly unrelated changes
- [x] Pinged someone for a review!

### CODE REVIEW

File: packages/cli/test/functional/test_site/expected/index.html
Comment (ang-zeyu) [inline packages/cli/test/functional/test_site/expected/index.html:200]: The test should be 
```
<md>
...
</md>
```

**Comment by damithc** [comment]
Thx for this PR @wxwxwxwx9 Pls ping @ang-zeyu when this is PR is ready for review

**Comment by wxwxwxwx9** [comment]
@damithc alright Prof! Will make a release later today after this is merged!

**Review by ang-zeyu — APPROVED** [review]
Lgtm 👍. Let's wait for #1601 as well if able, but a release without it is fine as well.

**Pull request merged**
